h/models/group.py

```diff
@@ -159,6 +159,27 @@ class Group(Base, mixins.Timestamps):
 
     pre_moderated: Mapped[bool | None] = mapped_column()
 
+    #: A custom Reply-To email address to be used for emails sent from this
+    #: group, for example: 'My Group Moderators <mygroup@example.com>'.
+    #:
+    #: This is used for example as the Reply-To address when the group's
+    #: moderators approve or decline an annotation in the group and we send
+    #: a notification email to the annotation's author.
+    reply_to = sa.Column(sa.UnicodeText(), nullable=True)
```

Comment:
If we are going to use this only for moderation emails, perhaps we should name it `moderation_reply_to`, otherwise, it may seem like other notifications that happen in this group should/may also be affected by this.

Notifications like mentions or replies could apply here, and in fact, we might also want to use this same email as the `Reply-To` for those.